frontend/src/features/projects/common/forms/AdditionalPropertyInformationForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { Form, FastInput } from 'components/common/form';
+import { useFormikContext } from 'formik';
+
+interface IAdditionalPropertyInformationFormProps {
+  isReadOnly?: boolean;
+}
+
+/**
+ * Surplus Properties information - in addition to standard property information.
+ * @param param0 isReadOnly disable editing
+ */
+const AdditionalPropertyInformationForm = ({
```

Comment:
This is a common form component used in both the SPL and ERP project information tabs.